src/libs/Navigation/Navigation.ts

```diff
@@ -129,7 +129,9 @@ function getDistanceFromPathInRootNavigator(path?: string): number {
             break;
         }
 
-        const pathFromState = getPathFromState(currentState, linkingConfig.config);
+        // When paths are compared, they the policyID param should be excluded from the state. If it's present in currentState, pathFromState will include it and in result it will be different than the path value.
```

Comment:
`they the ` please fix comment.
I would also shorten the comment a bit, I think its hard to parse because it has a lot of details. Just make it a little bit simpler